src/Client/Response/Base/Response.php

```diff
@@ -52,7 +52,7 @@ class Response
     /**
      * Array detailing which parameters were invalid.
      *
-     * @var array[]
+     * @var array<int, string>
```

Comment:
unsure about this array structure @slunak

Reply:
This is just an array of strings.

Example:

`{"user":"invalid","errors":["user identifier is invalid"], "status":0,"request":"5042853c-402d-4a18-abcb-168734a801de"}`

https://pushover.net/api#response

Reply:
Then it is correct 👍🏻

Reply:
`array<string>` would also work, right?

Reply:
Or string[]